src/schemas/general/Job.ts

```diff
@@ -5,4 +5,11 @@ export const JobSchema = z
     required_error: 'Job name is required',
   })
   .min(1, { message: 'Job name is required' })
-  .max(128, { message: 'Job name must be 128 characters or less' });
+  .max(128, { message: 'Job name must be 128 characters or less' })
+  .superRefine((value, ctx) => {
+    if (value.includes("'") || value.includes(',')) {
```

Comment:
This would not be easy to figure out, but could there be other characters that cause problems besides these two?

Reply:
I noticed it also fails with double quotes, so added extra validation for it.
![image](https://github.com/user-attachments/assets/d0ca5e06-918a-4611-aabe-8c3a0821b59c)
![image](https://github.com/user-attachments/assets/cb8ea2f0-4242-428f-8423-731e12060637)

Tested it with other special characters and it seems to work fine, even backticks.
![image](https://github.com/user-attachments/assets/605f68c8-89f5-47a3-ac5c-12f4375c1675)
![image](https://github.com/user-attachments/assets/9bbabf68-28c9-4234-8f44-185f25118056)

Reply:
We just found another case that breaks: when there is leading whitespace.  A job name of "   ABC" will get converted to a query for "ABC" at somepoint (somewhere along the way, part of the system is stripping the leading whitespace), and return no results. 

So we might want to also disallow leading or trailing whitespace (or strip it out automatically before we create the check).